widget/playground/src/containers/StyleLayout/StyleLayout.Preset.tsx

```diff
@@ -78,15 +77,15 @@ export function Preset(props: PresetTypes) {
     ? EACH_COL_HEIGHT * 2
     : EACH_COL_HEIGHT * (isAutoTab ? presetsSize : Math.ceil(presetsSize / 2));
 
-  const onSelectPreset = (customTheme: {
-    dark: WidgetColors;
-    light: WidgetColors;
-  }) => {
-    onSelectTheme(customTheme);
-    setSelectedPreset(customTheme);
+  const onSelectPreset = (customTheme: ColorsType) => {
+    const selectedTheme = isAutoTab ? customTheme : { [tab]: customTheme[tab] };
+    onSelectTheme(selectedTheme);
+    setSelectedPreset(selectedTheme);
     onChangeTheme({ name: 'singleTheme', value: isAutoTab ? undefined : true });
     if (!isAutoTab) {
```

Comment:
This if and else if pointless

```suggestion
onChangeTheme({ name: 'mode', value: tab });
```